<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineX1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineY1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineX2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>520</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>550</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineY2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>550</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLinea">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>520</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LineaDDA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnColor">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>550</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>500</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>3</width>
     <height>500</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>500</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblCoordenadas1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>P1 ( , )</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCoordenadas2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>100</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>P2 ( , )</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBresenham">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>520</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bresenham</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
